schema/network.yaml

```diff
@@ -68,6 +68,12 @@ definitions:
         type: string
         readOnly: true
         description: Indicate the interface name.
+      status:
+        type: boolean
+        description: Connection status of the interface, true for connected.
+      wan:
+        type: boolean
+        description: Indicate the interface worked as WAN or not (LAN).
```

Comment:
Route bundle depends on all network interface bundles, so it "will" loose all interfaces' information if it restart. The network manager we are thinking about is still under thinking... 
We could discuss tomorrow for better method.